pydruid/query.py

```diff
@@ -172,6 +172,10 @@ def export_pandas(self):
                 nres = [list(v['event'].items()) + [('timestamp', v['timestamp'])]
                         for v in self.result]
                 nres = [dict(v) for v in nres]
+            elif self.query_type == "scan":
+                nres = []
+                for item in self.result:
+                    nres += [e for e in item.get('events')]
```

Comment:
Nit, this can be a single line:

```python
nres = [e for item in self.result for e in item.get('events')]
```

Reply:
running out of line width :/, will keep as is